packages/core/package.json

```diff
@@ -3,32 +3,33 @@
   "version": "2.0.0-alpha.2",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
-    "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config webpack.config.dist.js",
-    "cs-check": "prettier -l \"{playground,src,test}/**/*.js\"",
-    "cs-format": "prettier \"{playground,src,test}/**/*.js\" --write",
-    "dist": "npm run build:lib && npm run build:dist",
-    "lint": "eslint src test playground",
+    "build": "npm run build:cjs && npm run build:es",
+    "build:cjs": "NODE_ENV=production BABEL_ENV=cjs babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/cjs",
+    "build:es": "NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/es",
```

Comment:
Why no umd, is there still a umd build? as this is what I am using! basically I include the whole lib and allow other plugins to use version pulled in via umd with requirejs, potentually umd works with both commonjs loaders, iife i.e. the browser straight and AMD loaders

Reply:
Not sure, any thoughts @chanceaclark ?

Reply:
I think we might have to change build tools then. `babel` has a plugin for converting to UMD but it's kind of a all or nothing approach.

@Saulzi, if you mention umd works with commonjs (cjs) loaders, couldn't you just load in the cjs modules?

In the meantime, play around with using rollup and try to push a commit.

Reply:
Yes it works with both commonjs and AMD loaders please refer to https://www.davidbcalhoun.com/2014/what-is-amd-commonjs-and-umd/
https://riptutorial.com/javascript/example/16339/universal-module-definition--umd-
https://www.google.com/search?q=common+javascript+modules&oq=common+javascript+modules&aqs=chrome..69i57.11167j0j7&sourceid=chrome&ie=UTF-8

AMD has been around for some time before the ES6 (ESM) standard.. i.e. dojo framework had amd support in around since 2011, many javascript developers have used things such as jquery etc which have used this pattern for some time and not even realised.. for example react builds to a umd module other popular javascript frameworks such as leaflet, moment, knockout and chart.js are built to umd modules.

@chanceaclark from experience rollup will make porting to typescript much easier...
as in my mind webpack should be taken out back and shot (have you ever seen the intro to stealjs video), crazy amount of configuration etc and the whole node_modules black hole. I will say that steal is good for sites but not libs and another potential is parcel.

Reply:
I've updated it to use the old webpack config to create the UMD build again. We can definitely switch to rollup but for now I'm just adding the old webpack config which works and won't affect local development.